core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -250,6 +250,13 @@ public void activateEndToEndWithTypedAudienceInt() throws Exception {
         eventHandler.expectImpression(activatedExperiment.getId(), actualVariation.getId(), testUserId, testUserAttributes);
     }
 
+    @Test
+    public void testOptimizelyConfig() throws Exception {
+        Optimizely optimizely = optimizelyBuilder.build();
+        OptimizelyConfig optimizelyConfig = optimizely.getOptimizelyConfig();
+        assertNotNull(optimizelyConfig);
+    }
+
```

Comment:
Can we add a test case for returning null for config error just like other APIs?